client/components/Header.tsx

```diff
@@ -0,0 +1,20 @@
+import back_arrow from "../images/back_arrow.png";
+import Image from "next/image";
+import Link from "next/link";
+
+export default function Header( {text, homePath}: Prop ){
```

Comment:
The header is just a general component. The back button should "pop the screens stack" and show the previous one. If the current site is opened from the home the back leads to the home. If it is opened from navigation the back leads to navigation.

I would suggest renaming "homePath" with "backPath" because the back button does not necessarily lead to the home (that's only an assumption I mentioned earlier).